flask_jwt_router/oauth2/google.py

```diff
@@ -245,7 +245,7 @@ def _set_expires(self):
         """
         self.expires_in = 3600 * 24 * 7
 
-    def create_test_headers(self, *, email: str, entity=None, scope="function") -> Dict[str, str]:
+    def create_test_headers(self, *, email: str, entity=None, scope="function") -> str:
```

Comment:
This looks to me like a breaking change? Was this all that was required for this PR?

Reply:
could you add the 2 other changes : 
https://github.com/joegasewicz/flask-jwt-router/issues/192
It's not a breaking change , this has already been released.